test/tbb/test_concurrent_queue.cpp

```diff
@@ -199,3 +200,50 @@ TEST_CASE("Test exception in allocation") {
 }
 
 #endif // TBB_USE_EXCEPTIONS
+
+struct TrackableItem {
```

Comment:
Could you please add new test that will test amount of constructed and destroyed items and test it with dtor and clear directly?

Reply:
Added dtor testing into the test with TrackableItem